### ISSUE TITLE
Provisioning profile "..." doesn't include the aps-environment entitlement. (in target '<my extension'> from project '<my project>'

### ISSUE THREAD

**Issue by mozzius** [andrew-levy/react-native-safari-extension#18]
Hi! I'm getting this error trying to make a local build:

```
error: Provisioning profile "*[expo] dev.mozzius.graysky.OpenInGrayskyExtension AppStore 2023-12-07T16:48:33.829Z" doesn't support the Push Notifications capability. (in target 'OpenInGrayskyExtension' from project 'Graysky')
error: Provisioning profile "*[expo] dev.mozzius.graysky.OpenInGrayskyExtension AppStore 2023-12-07T16:48:33.829Z" doesn't include the aps-environment entitlement. (in target 'OpenInGrayskyExtension' from project 'Graysky')
```

I do want to use expo-notifications in my main app, but it seems this breaks the safari extension. Looking at the code, it seems the config plugin does try to delete the aps-environment entitlement - not sure if that's just not working, or if something else is going on.

**Comment by mozzius** [comment]
Let me try and add it in - I haven't tried modifying anything yet

**Comment by mozzius** [comment]
patching the `withRemoveApsEntitlement` back in does not seem to fix it, unfortunately :(

**Comment by andrew-levy** [comment]
Thanks for checking :/  I'm not able to reproduce it on my end, so I'm not sure what the fix would be. My parent app isnt using push notifications, so that might be why? Will continue to look into this.

**Comment by mozzius** [comment]
I'll see what happens when I remove push notifications from mine

**Comment by andrew-levy** [comment]
I appreciate the testing help here! I have one more thing for you to try if you have the time. Try removing line 24 in this file: https://github.com/andrew-levy/react-native-safari-extension/blob/main/plugin/src/xcodeSafariExtension/addXCConfigurationList.ts#L24

```ts
CODE_SIGN_ENTITLEMENTS: `${appName}/${appName}.entitlements`,
```

This is pointing the extension target's entitlements to the parent app's entitlements. I just tried a local and eas build with this line removed, and both succeeded.

**Comment by mozzius** [comment]
Removing that line does not seem to help unfortunately, but I confirmed that removing expo-notifications allowed it to build correctly

**Comment by andrew-levy** [comment]
Interesting... I added expo-notifications and enabled push notifications and am seeing it fail with the same error as yours when line 24 is uncommented, and succeeds when its commented out. So it looks like that line has something to do with it. What build command are you running? I'm using `npx eas-cli build -p ios --profile ` and am using patch-package to make sure the change is being made in the library.

**Comment by mozzius** [comment]
Hey - my bad, it does work! I'd not changed the file structure of the extension after updating to v1, so fixing that (and patching the plugin) worked.

**Comment by andrew-levy** [comment]
No worries!! Awesome, thanks again for your help. I'll push out an update for that later today!

**Comment by andrew-levy** [comment]
Should be fixed in v1.0.2. Closing!
